File: 240417/정수 명령 처리 5/process-numeric-commands-5.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        
        ArrayList<Integer> arrList = new ArrayList<Integer>();
        Scanner s = new Scanner(System.in);

        int n = s.nextInt();

        for(int i = 0; i <n; i++){
            String cmd = s.next().toString();

            if(cmd.equals("push_back")){
                arrList.add(s.nextInt());
            }else if(cmd.equals("pop_back")){
                arrList.remove(arrList.size() - 1);
            }else if(cmd.equals("size")){
                System.out.println(arrList.size());
            }else if(cmd.equals("get")){
                System.out.println(arrList.get(s.nextInt() - 1));
            }

        }
    }
}

File: 240417/정수 명령 처리 5/process-numeric-commands-5.kt
import java.util.*

fun main(){
    val s = Scanner(System.`in`)

    val n = s.nextInt()

    val arrList = ArrayList<Int>()

    for (i in 1 .. 9){
        val cmd = s.next().toString()

        when(cmd){
            "push_back" -> { arrList.add(s.nextInt()) }
            "pop_back" -> { arrList.removeAt(arrList.size - 1) }
            "get" -> { println(arrList[s.nextInt() - 1]) }
            "size" -> { println(arrList.size) }
            else -> { }
        }
    }
}